src/libraries/System.Private.CoreLib/src/System/Runtime/ExceptionServices/InternalCalls.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+//
+// This is where we group together all the internal calls.
+//
+
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime
```

Comment:
```suggestion
namespace System.Runtime.ExceptionServices
```
to match the file path?